config/dockerfiles/ansible/Dockerfile

```diff
@@ -1,20 +1,32 @@
 FROM fedora:28
 
 LABEL maintainer="rnester@redhat.com, cmays@redhat.com, ari@redhat.com"
```

Comment:
LABEL name="contrainfra/ansible-executor" \
      maintainer="rnester@redhat.com, cmays@redhat.com, ari@redhat.com" \
      version="0.0.4" \
      description="A container to configure resources and run test scripts"